converged_sound_open_fw_modules_development/FW/intel_common/module_binmaps/aca_module.binmap

```diff
@@ -0,0 +1,35 @@
+module o ACA
+uuid 88881111-2222-3333-4444-555566667777
+name Aca module example
+version_major 0x1
+version_minor 0x0
+version_hotfix 0x0
+version_build 0x0
+affinity_mask MASTER_CORE_AFFINITY
+instance_count +1
+domain_types DP
+type AudClassModule
+stack_size 1000
+
+group text
+section .aca_module.text
+section ep .aca_module.cmi.text
+group rodata
+section .aca_module.rodata
+group bss
+section .aca_module.noload
+
+sched_caps 1 all
+
+mod_cfg 0 0 0 0 4096 500000 180 180 0 5000 0
```

Comment:
what do these fields mean - it best to have comments.

Reply:
It is described also in README - ```Module config``` section.